frontend-public/src/actionCreators/staticContentActionCreator.js

```diff
@@ -1,32 +1,2 @@
-import axios from "axios";
-import { notification } from "antd";
-import { showLoading, hideLoading } from "react-redux-loading-bar";
-import { request, success, error } from "@/actions/genericActions";
-import * as reducerTypes from "@/constants/reducerTypes";
-import * as staticContentActions from "@/actions/staticContentActions";
-import * as String from "@/constants/strings";
-import * as API from "@/constants/API";
-import { ENVIRONMENT } from "@/constants/environment";
-import { createRequestHeader } from "@/utils/RequestHeaders";
-
 // This file is anticipated to have multiple exports
 // eslint-disable-next-line import/prefer-default-export
```

Comment:
It looks like this file can just be removed. This comment is just there to prevent the eslint rule from barking about having a single non-default export.